docs/src/content/docs/ja/intro.md

```
@@ -31,26 +31,26 @@ Holloの特徴は、[フェディバース]（fediverse）の一部であるこ
 機能
 ----
 
-Holloは、ほとんどの個人ユーザーが必要とするものについては、Mastodonと同等の機能セットを提供します。
-期待できる主要なマイクロブログ機能をすべて提供する一方で、
+Holloは、ほとんどの個人ユーザーが必要とするものについて、Mastodonと同等の機能を提供します。
+期待できる主要なマイクロブログ機能をすべて利用できる一方で、
 （モデレーションツールなど）1人ユーザーインスタンスには必要ない機能は除外されています。
 つまり、次のようなことができます：
 
- -  ハッシュタグと言及
+ -  ハッシュタグとメンション
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)